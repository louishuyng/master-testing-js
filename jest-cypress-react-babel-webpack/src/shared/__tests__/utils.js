import {getFormattedValue} from '../utils'

test('formats the value', () => {
  expect(getFormattedValue('123.0')).toMatchSnapshot()
  expect(getFormattedValue('123.0')).toMatchInlineSnapshot(`"123.0"`)
})
