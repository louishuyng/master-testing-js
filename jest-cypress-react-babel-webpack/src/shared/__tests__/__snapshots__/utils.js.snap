// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`formats the value 1`] = `"123.0"`;
